recipes/rcat/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "RCAT" %}
+{% set version = "v1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jplind79/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c539ba7d0030119e47a0376859a3e670976a1aa4a6061f26480826dd4a113545
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - esmpy
+    - xesmf
+    - dask
+    - netcdf4
+    - dask-jobqueue
+
+test:
+  imports:
+    - rcat
+
+about:
+  home: https://github.com/jplind79/RCAT
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Regional Climate Analysis Tool (RCAT)'
+
+  description: |
+    RCAT is an analysis tool primarily developed for analysis of regional
+    climate models, but may also be used for global models as well. It's purely
+    written in Python where we aim for a modular code in a functional style.
+    The purpose is to get an efficient and structured way of collaboration
+    within our model developers but also for non pythonists that want to use
+    the tool for standard climate data analysis.
+  doc_url: https://regional-climate-analysis-tool.readthedocs.io
+  dev_url: https://github.com/jplind79/RCAT
+
+extra:
+  recipe-maintainers:
+    - jplind79
```

Comment:
@jplind79 Are you happy to be listed as a recipe maintainer here?

Reply:
Yes, I am.